packages/fxa-content-server/app/scripts/templates/force_auth.mustache

```diff
@@ -37,16 +37,16 @@
         {{#isPocketClient}}
           {{#unsafeTranslate}}
             By proceeding, you agree to:<br />
-            Pocket’s <a class="link-grey" id="pocket-tos" href="https://getpocket.com/tos/">Terms of Service</a> and <a id="pocket-pp" class="link-grey" href="https://getpocket.com/privacy/">Privacy Notice</a><br />
-            Firefox’s <a class="link-grey" id="fxa-tos" href="/legal/terms">Terms of Service</a> and <a class="link-grey" id="fxa-pp" href="/legal/privacy">Privacy Notice</a>
+            Pocket <a class="link-grey" id="pocket-tos" href="https://getpocket.com/tos/">Terms of Service</a> and <a id="pocket-pp" class="link-grey" href="https://getpocket.com/privacy/">Privacy Notice</a><br />
```

Comment:
I don't see in the ticket that it notes changing `Pocket’s` to `Pocket` or `Firefox’s` to `Mozilla Accounts` - am I missing that somewhere?

Reply:
I standardized - there was a discrepancy between backbone and react. I checked back through Slack conversations in moz-account-ux, and the terms should follow this format:
> By proceeding, you agree to the:
> Pocket Terms ...
> Mozilla Accounts Terms ...

Not sure how we missed the mentions of 'Firefox', but those do link to Mozilla Accounts Terms/Privacy so should be updated.

Thanks for catching the missing "the", I'll update!

Reply:
The joy of having repeatable stuff not in a partial/component 🥴

Reply:
Which is why the new React version has a standalone component for terms 🙌